functions.php

```diff
@@ -265,8 +265,17 @@ function display_meetings( $meetings ) {
 
 function display_meetings_by_year( $years ) {
 	ob_start();
```

Comment:
This `ob_start()` should be moved within the `if ( ! $years ):` if-statement.

Reply:
I need the output buffer open should the if statement be false. I could reopen it after the endif, but there shouldn't be a situation where it isn't closed properly the way it is. Thoughts?

Reply:
Ah ok.  In that case, you'd need to remove the `ob_start()` on this line: https://github.com/UCF/BOT-Bootstrap/pull/14/files#diff-78cd5aa3783a74555c9938a2a81d01c6R558  (otherwise you run the risk of opening multiple nested output buffers)

Reply:
Ignore previous comment...code is good.